packages/react-api/__tests__/api/model.test.js

```diff
@@ -51,6 +52,7 @@ describe('model', () => {
             type: 'query',
             source: longQuerySource.data,
             params: JSON.stringify(DEFAULT_PARAMS),
+            queryParameters: '',
```

Comment:
You are testing the 'empty' use case. But I think we should add some tests to see how it deals with different variations of real queryParameters. Probably it would be nice to have even 1 test variation per provider, so we are sure we  pass parameters as expected

Reply:
I have added a new case with a source (query) with `queryParameters` and a _parameterized_ query https://github.com/CartoDB/carto-react/pull/461/commits/2b2047ecdb32aa6488e1cdf1674595bc68deac52